guides/common/assembly_using-external-databases.adoc

```diff
@@ -14,8 +14,11 @@ endif::[]
 
 To create and use external databases for {Project}, you must complete the following procedures:
```

Comment:
This appears to be a very common pattern:
```console
$ rg 'you must' | wc -l
188
````

While I agree with your suggestion, should it be tackled project wide?

Reply:
> While I agree with your suggestion, should it be tackled project wide?

That might be risky. Consider cases like "you must not". Removing "you must" in bulk would break sentences like that.

While we could easily exclude individual cases like these ^, there might be others. We would need to come up with an exhaustive list, and I don't know how we would go about that.

Reply:
Looks like that is just one instance:
```console
$ rg 'you must' | grep -v 'you must not' | wc -l
187
```

In this case I can take care of it, but I'm cautious about blowing up the scope too much.